### ISSUE TITLE
Add "open" as option in link popup

### ISSUE THREAD

**Issue by arnowelzel** [nextcloud/collectives#1226]
**Is your feature request related to a problem? Please describe.**
When I click a link in "view" mode the link is not opened but instead there is now a popup which only offers to copy the link:

So it is impossible to click a link and just open it this way. One has to open a new tab first, then paste the copied link to the URL field.

**Describe the solution you'd like**
The popup should not only contain "copy" but also "open" as an option.

**Describe alternatives you've considered**
Add an option for Collectives not to show a popup at all for "view" mode but just let the browser handle them. When clicking a link it will  be opened, when clicking it while holding [Ctrl] it will open in a new tab and so on.

**Additional context**

Clicking a link opens a popup and not the link, but the popup does not give the option to open the link, only "copy":

![image](https://github.com/nextcloud/collectives/assets/6613614/5e8186b7-277f-4a0e-b087-2599f0f9aa7b)

**Comment by arnowelzel** [comment]
Also see https://github.com/nextcloud/collectives/issues/1227 - the thoughts about usability issues when changing the behaviour of well known elements like links also applies here.

**Comment by mejo-** [comment]
@arnowelzel could you elaborate on when and where you see the link bubble without a preview? Could it be that you suppress the preview from being displayed with a custom CSS rule (I read that in another report by you :wink:)

If for some reason no preview can be displayed, you should see a fallback with the URL being displayed instead.

Instead of adding an "open" button I'd like to eliminate all reasons why nothing is displayed in the area where the preview should be displayed.

**Comment by arnowelzel** [comment]
Yes, there is a fallback - but still very confusing, since you do not understand what this means - do I have to click again or not?

![grafik](https://github.com/nextcloud/collectives/assets/6613614/4169f760-a71a-41f7-afff-3ff9290c2eed)

And with "real" previews it is even more confusing - what do I have to do to open the link I just clicked?

![grafik](https://github.com/nextcloud/collectives/assets/6613614/90a6732d-6e1a-4a8f-b3b8-7a781ffec22d)

My main issue here is - in **view** mode do not open a popup at all! Why having a popup when clicking a link instead of just opening the link? The whole behaviour really confusing. People are used to the behaviour that a link will just open when clicking it. Just in editing mode it is useful to have a popup, since you **edit** a document and you may want to decide what to do with a link when **editing** the text. But when **viewing** it, just leave the behaviour as it is!

**Comment by nicolashimmelmann** [comment]
I agree with everyone who says they want a link to be a link in view mode. Where is the added value of the popup? I don't see any, it just makes life way more complicated.

For "normal" non-developer users, if they see a link and click it, they expect the content to open. Instead, they are presented with a popup which displays the same content (i.e. name of the page, or file or whatever) twice. And in this popup, it isn't even clear where to click to finally open the desired resource. This is a really really bad UX decision. 

I was trying to use a Collectives page as an index to quickly access certain folders in Files app. I want to access them as quickly as possible through one click, and not have to think several seconds about where I need to click in that popup. Please fix that :)

**Comment by okworx** [comment]
Here's another example when linking a (Nextcloud) poll on the same instance:

![Image](https://github.com/user-attachments/assets/0515aa9c-10a6-443d-bbc1-db3507a9ed3a)

To open this link, one needs to right-click and open it in a new tab or copy and paste it into their browser's address bar. This is a terribe UX for regular users who are used to just click a link in order to follow it (as it works in any other wiki btw). We have to add an explanation to this link to explain how people can open it. This shouldn't be necessary.
